src/core/shape_inference/include/squeeze_shape_inference.hpp

```diff
@@ -90,6 +90,18 @@ std::vector<TRShape> shape_infer(const Squeeze* op,
                              });
             }
         } else {
+            if (op->get_pytorch_dynamic_rank()) {
+                int i{};
```

Comment:
```suggestion
                int64_t i{-1};
```
Should be same type as `unique_axes` elements.

Reply:
done